src/component/TopStories.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react';
+import * as api from '../api/api';
+
+/**
+ *
+ *
+ * @class TopStories
+ * @extends {Component}
+ */
+class TopStories extends Component {
+
+  /**
+   * Creates an instance of TopStories.
+   *
+   * @memberof TopStories
+   */
+  constructor() {
+    super();
+    this.state = {
+      postIdList: []
+    };
+  }
+
+  componentDidMount = () => {
+    const storiesIndexPromise = api.getStoriesIndex('topstories');
```

Comment:
Why do we need to store the promise to a variable? Can't we just do the following?
```
api.getStoriesIndex('topstories')
  .then((req, res) => {
        this.setState({
          postIdList: req.data
        });
      })
      .catch();
```